src/bci_build/package/templates/access_protected_images.j2

```diff
@@ -0,0 +1,37 @@
+## Accessing the Container Image
+
+This Container Image requires an subscription from SUSE to access. In order
```

Comment:
```suggestion
This Container Image requires a subscription from SUSE to access. In order
```

Reply:
I've updated it to the wording in the container guide.